src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid $gray-elements-color;
+  border-radius: 5px;
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-size: 12px;
+    color: $main-accent-color;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__code {
+    margin-top: 4px;
+    color: $secondary-color;
+    font-size: 10px;
+    line-height: 14px;
+    align-self: flex-start;
+  }
+
+  &__score {
+    margin-top: 16px;
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    color: $main-accent-color;
+    text-align: right;
+    font-size: 10px;
+    align-self: center;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+
+    &-label {
+      font-size: 12px;
+      line-height: 18px;
+      font-weight: 400;
+      color: $secondary-color;
+    }
+
+    &-value {
+      font-size: 16px;
+      line-height: 18px;
+      font-weight: 700;
+      color: $main-accent-color;
+    }
+  }
+
+  &__button {
+    margin-top: 16px;
+    background-color: $blue-accent-color;
+    color: #fff;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    border-radius: 5px;
+    line-height: 40px;
+    text-decoration: none;
+
```

Comment:
please add here border 1px solid transparent or white, because right now on button hover your layout is bouncing :) 

check:

![Screenshot 2023-09-07 at 07 40 54](https://github.com/mate-academy/layout_catalog/assets/30219274/d433de2f-4602-4835-af29-16b08f82ee88)